imageroot/actions/alter-share/50alter_share

```diff
@@ -20,9 +21,12 @@ if pcheck.returncode != 0:
     json.dump([{'field':'name', 'parameter':'name','value': request['name'], 'error':'share_not_found'}], fp=sys.stdout)
     sys.exit(2)
 
-samba_add_share = podman_exec + ["net", "conf", "setparm", request["name"]]
-
 if 'description' in request:
-    samba_add_share += ["comment", request["description"]]
-
-phelper = agent.run_helper(*samba_add_share).check_returncode()
+    set_description_cmd = podman_exec + ["net", "conf", "setparm", request["name"], "comment", request["description"]]
+    agent.run_helper(*set_description_cmd)
```

Comment:
This command prints the share configuration to the shell when using `api-cli` command: it seems an error to the user. Can we avoid it?